core/pta/tcta.c

```diff
@@ -1,5 +1,9 @@
 
 #include <kernel/pseudo_ta.h>
+#include <mm/core_memprot.h>
```

Comment:
Don't think you need all these. Just mm/core_mmu.h is enough

Reply:
Done. Changed  to <mm/core_mmu.h>